### PR TITLE
Fix for issue

### PR DESCRIPTION
**Pull request by washingdone**
Fixes issue #23

### CODE REVIEW

File: src/Command.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ class Command {
       .then(msg => {
         this.self.counts.msgsSent = this.self.counts.msgsSent + 1
         if (deleteDelay) {
-          if (!this.deleteAfter || !this.config.deleteCommandMessages) return resolve(msg)
+          if (this.deleteAfter) return resolve(msg)
```
Comment (TheRacingLion) [inline src/Command.js:93]: Hey @washingdone. If this is tested and you confirmed it works, can you apply this fix to the rest of the code lines where `if (!this.deleteAfter || !this.config.deleteCommandMessages) return resolve(msg)` appears?
If I am not wrong the only places where that happens are [here](https://github.com/TheRacingLion/Discord-SelfBot/blob/master/src/Command.js#L73), [here](https://github.com/TheRacingLion/Discord-SelfBot/blob/master/src/Command.js#L93) and [here](https://github.com/TheRacingLion/Discord-SelfBot/blob/master/src/Command.js#L115). (The second one behind the line that is fixed in here)
Would be a great help.

**Review by TheRacingLion — CHANGES_REQUESTED** [review]

File: src/Command.js (hunk 1 above)
Comment (washingdone) [reply src/Command.js:93]: all lines should be fixed

**Review by TheRacingLion — APPROVED** [review]

**Comment by TheRacingLion** [comment]
Thanks for the PR. ;)

**Comment by washingdone** [comment]
you're welcome

**Pull request merged**